packages/ocean-components/src/index.ts

```diff
@@ -24,3 +24,6 @@ export * from './TextArea';
 
 export { default as Select } from './Select';
 export * from './Select';
+
+export { default as Switch } from './Switch';
+export * from './Switch';
```

Comment:
Adds a line at the end of the file.

```suggestion
export * from './Switch';

```